### ISSUE TITLE
Zero Models Showing In Subscriptions List

### ISSUE THREAD

**Issue by RandomGuy5598** [UltimaHoarder/UltimaScraper#2026]
I'm not getting any subscriptions showing up when I run the scraper. It was working perfectly until about 2-3 weeks ago, then this issue started. I saw everything moved to this account and that some stuff had been changed. I uninstalled everything, downloaded the code and ran the updater. Everything went smoothly, no issues. But when I run the scraper I still get the same issue as before, I get zero models in my subscriptions list.

I've checked and double-checked my auth info and it's all good. Anyone have any insights or tips on what to try? Any help would be greatly appreciated.

Thanks!

**Comment by airigames12** [comment]
Same happening here

**Comment by anthonyphua** [comment]
Check in your config file for the following 

"active": true
"id": XXXXXXXX (This is the same 8 digits as per the auth_id in the cookie),

**Comment by RandomGuy5598** [comment]
> Check in your config file for the following
> 
> "active": true "id": XXXXXXXX (This is the same 8 digits as per the auth_id in the cookie),

In the config file, I see the "active": under "tui", and I changed that to true. For clarity, I'm assuming that's the correct one, as that's the only one in the default config that was created. There is no "id": in the config file that was created. However, there is an "id": in the auth file that was defaulted to null. So, I changed null to the same 8-digit number from auth_id and it still does the same thing.

Was there supposed to be a line with "id": in the config file that wasn't in mine when it was created from the initial setup?

Thanks for the reply. I really appreciate the help, and anymore help available. :)
Cheers!
